packages/binarytree/tsconfig.prod.esm.json

```diff
@@ -2,9 +2,11 @@
   "extends": "../../config/tsconfig.prod.esm.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "dist/esm",
-    "composite": true
+    "outDir": "dist/esm"
```

Comment:
Do we need this in all the package specific `tsconfig.prod.esm.json` files or can we move this up to the `root/config/cli/tsconfig.prod.esm.json` since we're consolidating things?